src/pages/bridge/TransferDialog.tsx

```diff
@@ -112,7 +112,7 @@ interface TransferDialogProps {
 
 export function CompletedTransferDialog(props: TransferDialogProps) {
   const { transfer, visible, onClose } = props;
-  const stellarAsset = transfer.original.asset.asStellar.asAlphaNum4.code.toHuman()?.toString();
```

Comment:
What? Weird. I feel like I already fixed this, it should have been some conflict that removed it. But I clearly remember this line was replaced by a conversion to an asset code (I even think you reviewed that, but now I can't find the PR)

Reply:
Or maybe I'm wrong and it wasn't in the dialogs, anyway I have added a function decodeAssetCode() to bridge/helpers.ts that does a similar thing than currencyToString(), we might wanna use the same everywhere

Reply:
Yeah, I remember reading something similar not too long ago haha, but not sure when and where. 
Ah right, we should probably just use one if they serve the same purpose. I think `currencyToString()` is more powerful though since it also tries to map the XCM index to the proper name.